src/run_validation.py

```diff
@@ -10,19 +10,28 @@
 
 from system import console
 from validation_workflow.validation_args import ValidationArgs
+from validation_workflow.validation_test_runner import ValidationTestRunner  # type: ignore
 
 
 def main() -> int:
-
     args = ValidationArgs()
 
     console.configure(level=args.logging_level)
     logging.getLogger("urllib3").setLevel(logging.WARNING)
 
-    dist = args.DISTRIBUTION_MAP.get(args.distribution, None)
-    dist.download_artifacts(projects=args.projects, version=args.version)
-
-    return 0
+    dist = None
+    if args.distribution == 'docker':
+        dist = "docker"
+    elif args.distribution == "tar":
+        dist = "tar"
+    elif args.distribution == "yum":
+        dist = "yum"
+    elif args.distribution == "rpm":
+        dist = "rpm"
```

Comment:
Why cant dist being directly assigned with args.distribution since they are the same value anyway?

Reply:
Will take care of this Peter, thanks